src/teleport.js

```diff
@@ -0,0 +1,37 @@
+import { checkForAlpine } from './utils'
+
+const AlpineTeleportMagicMethod = {
+    start() {
+        checkForAlpine()
+        Alpine.addMagicProperty('teleport', ($el) => {
+            return function (target = document.body, $element = $el, prepend = false) {
+                // find the portal
+                const destination = document.querySelector(`[x-portal="${target}"]`)
+
+                // throw error if portal is undefined
+                if (destination === undefined) {
+                    throw Error('Undefined portal: ', destination)
```

Comment:
throwing error vs console.warn-ing?

Also it might be better to have the message say

```suggestion
                    throw Error(`Portal "${target}" not found, got "${destination}", does your element have the x-portal="${target}" attribute?`)
```

Reply:
$teleport not defaulting to `body` if `target` is provided. So warning didn't make sense to me. But we can change if all agrees. 

about message yeah, i agree. Will change now.